SecurityCardManagement/README.md

```diff
@@ -36,10 +36,9 @@ Before forwarding the request to a security card, a browser must request a pin c
 arguments: 
 
 	answerId: uint32 - function id of result callback	
-	sn	: uint192 - card serial number
 	p1      : uint128 - authentication password
 	iv      : uint16 - vector for AES128 CBC initialization for encrypting P1
-	ecs     : uint32 - encrypted common secret
+	cs     : uint32 - common secret
```

Comment:
Why is it not encrypted anymore? I believe this was made on purpose in order not to pass the plain unencrypted data via the interface

Reply:
yep, just common secret here